staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2723,8 +2723,25 @@ func describeService(service *corev1.Service, endpoints *corev1.Endpoints, event
 		w.Write(LEVEL_0, "Type:\t%s\n", service.Spec.Type)
```

Comment:
we don't change the `get` because scripts depend on column order and data. So we usually just update `describe`. This is what we followed before for `IPFamily` and `Pod.Status.PodIPs`